### PR TITLE
Do not show toasts for bucket errors but notify airbrake instead

### LINKED ISSUE #4819: Don't show bucket-failed-to-fetch warning immediately
For some users, the warnings appear quite often (every 20 min?). We should double-check that the warning only appears if some requests fail repeatedly.

### PR DESCRIPTION
**Pull request by daniel-wer**
For historical context, initially the toasts were enabled as part of https://github.com/scalableminds/webknossos/issues/4725.

Check airbrake and search for `requesting data` to see the error that is logged to airbrake.

### URL of deployed dev instance (used for testing):
- https://donotshowbucketerrors.webknossos.xyz

### Steps to test:
- Open a tracing on the deployed dev branch, open dev tools and choose "Offline" in the network tab. Move, so that buckets are requested. There should be no toast regarding the failed bucket requests. Check airbrake for the errors.

### Issues:
- fixes #4819 

------
- [ ] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes
- [ ] Needs datastore update after deployment
- [ ] Ready for review

### CODE REVIEW

File: frontend/javascripts/oxalis/model/bucket_data_handling/wkstore_adapter.js [hunk 1]
```diff
@@ -171,22 +171,19 @@ export async function requestFromStore(
...
-    let detailedError =
+    const detailedError =
       errorResponse.status != null
-        ? `Status code ${errorResponse.status} - "${errorResponse.statusText}".`
+        ? `Status code ${errorResponse.status} - "${errorResponse.statusText}" - URL: ${
+            errorResponse.url
+          }.`
         : errorResponse.message;
 
-    detailedError += `(URL: ${dataUrl})`;
```
Comment (daniel-wer) [inline frontend/javascripts/oxalis/model/bucket_data_handling/wkstore_adapter.js:179]: If the error is not a response with a status code, the requests lib already appends the URL which is why I moved this from here to the conditional above.

File: frontend/javascripts/libs/request.js [hunk 2]
```diff
@@ -266,13 +266,6 @@ class Request {
       setTimeout(() => resolve("timeout"), timeout);
     });
 
-  handleStatus = (response: Response): Promise<Response> => {
```
Comment (daniel-wer) [inline frontend/javascripts/libs/request.js:269]: This method was unused, `handle_http_status.js` is used everywhere, instead.

File: frontend/javascripts/oxalis/model/bucket_data_handling/wkstore_adapter.js [hunk 3]
```diff
@@ -171,22 +171,19 @@ export async function requestFromStore(
...
-    Toast.warning(
-      errorMessage,
-      {
-        key: errorMessage,
-        sticky: false,
-      },
+    ErrorHandling.notify(new Error(errorMessage), {
       detailedError,
-    );
+      isOnline: window.navigator.onLine,
```
Comment (philippotto) [inline frontend/javascripts/oxalis/model/bucket_data_handling/wkstore_adapter.js:185]: Cool beans! Didn't know about that.

**Review by philippotto — APPROVED** [review]
Awesome! :+1: for the additional clean up!

File: frontend/javascripts/oxalis/model/bucket_data_handling/wkstore_adapter.js [hunk 4]
```diff
@@ -171,22 +171,19 @@ export async function requestFromStore(
     const errorMessage = `Requesting data from layer "${
       layerInfo.name
```
Comment (philippotto) [inline frontend/javascripts/oxalis/model/bucket_data_handling/wkstore_adapter.js:172]: One thing I noticed: Having the layer name in the error message inhibits the aggregation in airbrake. Maybe we can omit the layer name when sending the message to airbrake?

Comment (daniel-wer) [reply frontend/javascripts/oxalis/model/bucket_data_handling/wkstore_adapter.js:172]: For reference, airbrake is able to figure out that the layer name is dynamic and abstracts from that, so the aggregation still works :+1:

**Pull request merged**